src/js/editor/editor.js

```diff
@@ -293,7 +306,7 @@ class Editor {
     this.run(postEditor => {
       let nextPosition = postEditor.deleteAtPosition(position, direction, {unit});
       postEditor.setRange(nextPosition);
-    });
+    }, EDITOR_RUN_LAST_ACTION_STATE.DELETE);
```

Comment:
I am reluctant to modify the method signature of the already-public `run` method in order to add a private-ish second parameter...